tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md

```diff
@@ -175,7 +180,10 @@ The `#[pallet::call]` macro defines the dispatchable functions (or calls) the pa
 The structure of the dispatchable calls in this pallet is as follows:
 
 ```rust
---8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet/call_structure.rs'
+--8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet/src/lib.rs:73:84'
```

Comment:
Can you share a screenshot of some of the rendering as well please? 

In general, one of the challenges of using real code might be that it might be harder to hide unrelated parts etc.

Reply:
Agreed, the way I am going about it now is to use snippets to select individual sections of code that we want to display. The downside is that to have correct indentations, I had to make adjustments in the `.md` files but I don't think that's an issue just something to look out for. I will add screenshots of the rendering.

Reply:
<img width="1440" alt="Screenshot 2024-12-31 at 17 53 00" src="https://github.com/user-attachments/assets/8eadcee3-1233-43de-b523-0476ee799234" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 53 13" src="https://github.com/user-attachments/assets/63999bcc-a9fd-4792-aa1d-b9edde27a818" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 53 24" src="https://github.com/user-attachments/assets/34f88848-a3bd-4a98-8466-1a138e5fa0e8" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 53 33" src="https://github.com/user-attachments/assets/64282796-a19e-44c3-8545-0e91990c5321" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 53 42" src="https://github.com/user-attachments/assets/006f24fa-bdbd-4106-ad9b-e06401ba2fb5" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 53 56" src="https://github.com/user-attachments/assets/de7f58e4-94dc-4579-a90b-4826c11a87bc" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 54 09" src="https://github.com/user-attachments/assets/30f692e3-f9cc-4d07-82d6-03ad9a1f8e42" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 54 20" src="https://github.com/user-attachments/assets/f7f9a106-aca3-427d-ad97-403b54e0323a" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 54 28" src="https://github.com/user-attachments/assets/5f73b13a-1178-4943-96b3-53c5a608b434" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 54 38" src="https://github.com/user-attachments/assets/f9a72522-05f4-4af0-853c-d3d585bd9993" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 54 46" src="https://github.com/user-attachments/assets/3df7748a-2cfc-4a02-9a2e-13aaa9bb9268" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 54 55" src="https://github.com/user-attachments/assets/71091f4a-0ccf-424a-b733-75f9f701d708" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 55 04" src="https://github.com/user-attachments/assets/cacbd3c4-d27e-476e-a08e-4310d569a8c1" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 55 12" src="https://github.com/user-attachments/assets/0050dbed-6982-4a8e-b67a-83862dbb5cc9" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 55 22" src="https://github.com/user-attachments/assets/8f3a3f80-1750-4387-9cca-e12601d596f6" />
<img width="1440" alt="Screenshot 2024-12-31 at 17 55 30" src="https://github.com/user-attachments/assets/01116610-c84d-4f2a-8849-fefbbdfaa175" />